tardis/tardis_portal/views.py

```diff
@@ -425,13 +425,33 @@ def datetime_to_us(dt):
         owners = parent_experiment.get_owners()
         datafiles = []
         dataset_size = 0
+        verified_datafiles_count = 0
+        verified_datafiles_size = 0
         for datafile in datafile_objects:
+            if datafile.verified:
+                verified = "Yes"
+                verified_datafiles_count += 1
+                verified_datafiles_size += int(datafile.size)
+            else:
+                verified = "No"
+                try:
+                    file_object_size = \
+                        datafile.file_object.size \
+                        if datafile.file_object else 0
+                    if file_object_size < int(datafile.size):
+                        verified = "No (%s of %s bytes uploaded)" \
+                            % ('{:,}'.format(file_object_size),
+                               '{:,}'.format(int(datafile.size)))
```

Comment:
Yes, that looks more elegant.  No there wasn't a good reason why I didn't write it that way.  Feel free to change it.